site/src/atomics/display.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Display Atomics
+description: Display related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Display Atomics
+
+Display atomics are used to set the display behavior of an element.
+
+| cssproperty | value                                                                    | screensize                        |
+| ----------- | ------------------------------------------------------------------------ | --------------------------------- |
+| `display`   | `block`, `flex`, `inline`, `inline-block`, `inline-flex`, `grid`, `none` | `tablet`, `desktop`, `widescreen` |
+
+## Usage
+
+Here is an example of display atomics being used.
+
+```html
+<div class="display-block border">
+	<div>Foo</div>
+	<div>Foo</div>
+</div>
+```
+
+### Responsive rules
+
+Here are a few examples of display atomics being used for different screensizes.
+
+```html
+<div class="display-flex-tablet border">
+	<div>Bar</div>
+	<div>Bar</div>
+</div>
+```
+
+```html
+<div class="display-inline-desktop border">
+	<div>Baz</div>
+	<div>Baz</div>
+</div>
```

Comment:
shouldn't the `display-inline-desktop` class be applied to the inner `div`'s?

Reply:
Ah yes, updated